app/waf/src/models/key.model.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2019 F5 Networks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CommonEntity} from '.';
+import {model, property} from '@loopback/repository';
+
+@model()
+export class Key extends CommonEntity {
+  @property({
+    type: 'string',
+    required: false,
+    schema: {
+      create: false,
+      update: false,
+      response: true,
+      example: '/Common/private.key',
+    },
+    as3: {},
+  })
+  file?: string;
+
+  @property({
+    type: 'string',
+    required: true,
+    schema: {
+      create: true,
+      update: false,
+      response: true,
+      required: true,
+      example: '/root/key.private',
+    },
+    as3: {},
+  })
+  localfile: string;
```

Comment:
What is the purpose to add this property?

Reply:
yes, this property is not much useful. Just record while local file we load the cert/key from.  will remove this.